spacy/tests/doc/test_json_doc_conversion.py

```diff
@@ -57,18 +99,116 @@ def test_doc_to_json(doc):
     assert json_doc["ents"][0]["end"] == 3  # character offset!
     assert json_doc["ents"][0]["label"] == "ORG"
     assert not schemas.validate(schemas.DocJSONSchema, json_doc)
+    assert srsly.json_loads(srsly.json_dumps(json_doc)) == json_doc
 
 
 def test_doc_to_json_underscore(doc):
-    Doc.set_extension("json_test1", default=False)
-    Doc.set_extension("json_test2", default=False)
+    if not Doc.has_extension("json_test1"):
```

Comment:
Instead of checking for the presence of extensions repeatedly, use this as an import and it will automatically clear after each function:

```
from .test_underscore import clean_underscore  # noqa: F401
```